src/init.cpp

```diff
@@ -820,12 +820,7 @@ bool AppInit2(Config &config, boost::thread_group &threadGroup, CScheduler &sche
     // BUIP010 Xtreme Thinblocks: begin section Initialize XTHIN service
     if (GetBoolArg("-use-thinblocks", DEFAULT_USE_THINBLOCKS))
         nLocalServices |= NODE_XTHIN;
-// BUIP010 Xtreme Thinblocks: end section
-
-// UAHF - BitcoinCash
-#ifdef BITCOIN_CASH
-    nLocalServices |= NODE_BITCOIN_CASH;
-#endif
+    // BUIP010 Xtreme Thinblocks: end section
 
     nMaxTipAge = GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
```

Comment:
I don't think we should be removing the bitcoin cash service bit just yet...i agree this needs to be sunsetted but first we need to remove only when BU nodes disconnect from nodes that do not present the bit.  Later , perhaps in a year, we should reclaim the bit.  This is what XT is doing and I believe is correct since there may be other nodes (older nodes for one that are expecting the bit) and posssibly other wallets or services that are still expecting the bit to be present.

Reply:
agreed. CASH service bit reinstated.